### PR TITLE
Upgraded ps compliance

### PR DESCRIPTION
**Pull request by nicosomb**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | 2.x
| Description?      | Upgraded ps compliance
| Type?             |  improvement
| BC breaks?        | no
| Deprecations?     | no
| Fixed ticket?     | 
| How to test?      | 
| Possible impacts? |

### CODE REVIEW

**Review by matks — APPROVED** [review]
Looks good to me 👍 waiting for MBO team approval

**Comment by sowbiba** [comment]
Thanks @nicosomb

**Pull request merged**
